src/main/webapp/js/all-section-deals.js

```diff
@@ -12,22 +13,51 @@ function createAllDealCards(deals) {
       }
     }
   }
+  const dropdownMenu = document.getElementById('sort');
+  if (reqSection !== 'trending') {
```

Comment:
I didn't understand why reqSection `trending` has separate behavior. Could you add a comment with some explanation?